src/pages/index.tsx

```diff
@@ -35,10 +37,14 @@ const Home: NextPage = () => {
             <div className="text-sm text-gray-500 mb-2">
               Try Parachute right now
             </div>            
-            <button onClick={() => void signIn("google")} className="bg-black text-white text-center font-semibold rounded-lg w-full p-3">
+            <button onClick={() => void signIn("google", { 
+                          callbackUrl: `${window.location.origin}/dashboard`,
+                  })} className="bg-black text-white text-center font-semibold rounded-lg w-full p-3">
               Sign in with Google
             </button>
-            <button onClick={() => void signIn("auth0")} className="border text-center font-semibold rounded-lg w-full p-3">
+            <button onClick={() => void signIn("auth0", { 
+                          callbackUrl: `${window.location.origin}/dashboard`,
+                  })} className="border text-center font-semibold rounded-lg w-full p-3">
```

Comment:
Good work on this! However, we should enforce redirect from `/` base URL to `/dashboard` if we detech user login. This way if they clicked `parachute.fyi` via a web link they'll still get redirected to `/dashboard`

Reply:
We may implement this after a while. #24